src/types/Filter.ts

```diff
@@ -0,0 +1,7 @@
+export enum FILTER {
+  ALL = 'all',
+  ACTIVE = 'active',
+  COMPLETED = 'completed',
+}
```

Comment:
```suggestion
export enum Filter {
  All = 'all',
  Active = 'active',
  Completed = 'completed',
}
```
Use Pascal Case instead of Upper Case for enum name and keys